stripper/ze_FORiUS_just_RUN_p2.cfg

```diff
@@ -10,4 +10,22 @@ modify:
 	{
 		"dmg" "999999"
 	}
-}
\ No newline at end of file
+}
+
+;Remove Weaponcleaner
```

Comment:
As I've told you before... there is zero point in removing a weapon cleaner on a map with no items. They are only removed to stop items from despawning.